models/DataObject/ClassDefinition/Dao.php

```diff
@@ -159,9 +159,9 @@ public function update()
         // add non existing columns in the table
         if (is_array($this->model->getFieldDefinitions()) && count($this->model->getFieldDefinitions())) {
             foreach ($this->model->getFieldDefinitions() as $key => $value) {
-                if ($value instanceof DataObject\ClassDefinition\Data\ResourcePersistenceAwareInterface || method_exists($value, 'getDataForResource')) {
+                if ($value instanceof DataObject\ClassDefinition\Data\ResourcePersistenceAwareInterface) {
                     // if a datafield requires more than one column in the datastore table => only for non-relation types
-                    /** @var Data&DataObject\ClassDefinition\Data\ResourcePersistenceAwareInterface $value */
+                    /** @var DataObject\ClassDefinition\Data\ResourcePersistenceAwareInterface $value */
```

Comment:
@brusch These comments can now also be removed. PHPStan and IDEs should be know it with the if statement above.

Reply:
👍thanks a lot!